internal/provider/provider_test.go

```diff
@@ -72,9 +97,17 @@ const (
 		data "kentik-cloudexport_item" "aws" {
 			id = "1"
 		}
-	`
-	invalidRetryConfig = `
+	`,
+		apiURL,
+	)
+}
+
+func makeInvalidRetryConfig(apiURL string) string {
```

Comment:
What is "invalid" about the retry config bellow?

Reply:
max_delay is invalid, set to: max_delay = "invalid-delay"

Reply:
I see.